projects/make-a-vis/src/lib/shared/services/load-project/load-project.service.ts

```diff
@@ -198,6 +230,40 @@ export class LoadProjectService {
       return extnObject.label === extensionLabel;
     }))[0]['extensions'].join(',');
   }
+
+  private confirmContinueLargeInputLoading(): Observable<boolean> {
+    const ref = this.dialog.open<ConfirmationDialogComponent, ConfirmationDialogOptions, boolean>(
+      ConfirmationDialogComponent,
+      {
+        // width, height?
+        closeOnNavigation: true,
+        disableClose: true,
+        data: {
+          title: 'Continue loading large dataset?',
+          content: '', // TODO
```

Comment:
Large datasets (> 5,000 records) may take a long time to load and use excessive memory.